cmd/autoheal/healer.go

```diff
@@ -149,6 +158,27 @@ func (b *HealerBuilder) Build() (h *Healer, err error) {
 
 	// initialize active jobs map:
 	h.activeJobs = new(syncmap.Map)
+
```

Comment:
@yaacov what about moving this logic from `Healer` to `Loader`? sounds to me like this logic is part of the `Loader`s job an could be cleaner there.
Meaning `Loader` should hold the observer and watch the files and not `Healer`.

Reply:
looking into it :+1:

Reply:
Done :+1: